Rubberduck.RibbonDispatcher/ControlMixins/ImageableMixin.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Runtime.CompilerServices;
```

Comment:
But they are NOT (just) extensions. These are Mixins, and that is an important distinction that the reader, and any future maintainer, must be made aware of. I am open to suggestions in how to properly emphasize the distinction, but I will not blindly change the name in a way that hides, instead of emphasizing, the distinction.